Jenkinsfile-insecure.groovy

```diff
@@ -4,6 +4,10 @@ dir("packages/adminrouter/extra/src") {
     }
 
     try {
+        stage('make check-api-docs') {
```

Comment:
Btw, we will also have to put it into EE: https://github.com/mesosphere/dcos-enterprise/blob/master/Jenkinsfile#L55-L64